ktor-server/ktor-server-host-common/jvmAndNix/src/io/ktor/server/engine/CommandLine.kt

```diff
@@ -130,6 +130,12 @@ public fun BaseApplicationEngine.Configuration.loadCommonConfiguration(deploymen
     deploymentConfig.propertyOrNull("workerGroupSize")?.getString()?.toInt()?.let {
         workerGroupSize = it
     }
+    deploymentConfig.propertyOrNull("shutdownGracePeriod")?.getString()?.toLong()?.let {
+        shutdownGracePeriod = it
+    } ?: { shutdownGracePeriod = 1000 }
```

Comment:
Could we extract default values to a global constant and use the same value everywhere?

Reply:
Thanks, I deleted the setting of default values here because `shutdownGracePeriod` and `shutdownTimeout` already have default values in [Configuration](https://github.com/ktorio/ktor/blob/a08b7c8f520ca41f588e00f8c0cee9a83d1e346e/ktor-server/ktor-server-host-common/jvmAndNix/src/io/ktor/server/engine/ApplicationEngine.kt#L49)